index.html

```diff
@@ -249,6 +249,23 @@ <h2>About Myself</h2>
                 </div>
             </div>
         </section>
+        <section class="form-section">
+            <h2>Contact me</h2>
+            <p>
+                If you have an application you are interested in 
+                developing, a feature that you need built or a 
+                project that needs coding. I’d love to help with it!
+            </p>
+            <form action="">
```

Comment:
-  [ ] You should use [Formspree](https://formspree.io) to submit the form , according to the project requirements